doc/release-notes/10943-featured-items.md

```diff
@@ -0,0 +1,8 @@
+CRUD endpoints for Collection Featured Items have been implemented. In particular, the following endpoints have been implemented:
+
+- Create a feature item (POST /api/dataverses/<dataverse_id>/featuredItems)
+- Update a feature item (PUT /api/dataverseFeaturedItems/<item_id>)
+- Delete a feature item (DELETE /api/dataverseFeaturedItems/<item_id>)
+- List all featured items in a collection (GET /api/dataverses/<dataverse_id>/featuredItems)
+- Delete all featured items in a collection (DELETE /api/dataverses/<dataverse_id>/featuredItems)
+- Update all featured items in a collection (PUT /api/dataverses/<dataverse_id>/featuredItems)
```

Comment:
```suggestion
- Update all featured items in a collection (PUT /api/dataverses/<dataverse_id>/featuredItems)

See also #10943 and #11124.
```

Having the issue and PR in the snippet makes writing the final release notes easier. Please see #10445.